samples/matter/basic/prj.conf

```diff
@@ -0,0 +1,101 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Enable CHIP
+CONFIG_CHIP=y
+CONFIG_CHIP_PROJECT_CONFIG="src/chip_project_config.h"
+CONFIG_STD_CPP14=y
+
+# General networking settings
+CONFIG_NETWORKING=y
+CONFIG_NET_IPV6_NBR_CACHE=n
+CONFIG_NET_IPV6_MLD=n
+CONFIG_NET_SOCKETS_POLL_MAX=5
+
+# Enable OpenThread MTD
+CONFIG_NET_L2_OPENTHREAD=y
+
+CONFIG_OPENTHREAD_MTD=y
+CONFIG_OPENTHREAD_FTD=n
+CONFIG_OPENTHREAD_SLAAC=y
+CONFIG_OPENTHREAD_MTD_NETDIAG=y
+CONFIG_OPENTHREAD_MANUAL_START=y
+CONFIG_OPENTHREAD_THREAD_STACK_SIZE=6144
+CONFIG_OPENTHREAD_NUM_MESSAGE_BUFFERS=64
+
+CONFIG_OPENTHREAD_PANID=4660
+CONFIG_OPENTHREAD_CHANNEL=15
+CONFIG_OPENTHREAD_NETWORK_NAME="OpenThread"
+CONFIG_OPENTHREAD_XPANID="11:11:11:11:22:22:22:22"
+
+# Enable Bluetooth Low Energy
+CONFIG_BT=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_DEVICE_APPEARANCE=0
+CONFIG_BT_DEVICE_NAME_DYNAMIC=y
+CONFIG_BT_DEVICE_NAME_MAX=15
+CONFIG_BT_DEVICE_NAME="MatterLock"
```

Comment:
I think that BT device name should be changed to something else - honestly I don't have idea how, cause advertised "MatterBasic" looks pretty weird for me.